src/components/navbar/Wallet.tsx

```diff
@@ -1,54 +1,93 @@
-import { AccentText } from 'components/Text'
+import { AccentText, SocialLink } from 'components/Text'
 import { useSnapshot } from 'valtio'
+import Discord from 'icons/Discord'
 import EnsAddress from 'components/EnsAddress'
 import SealWallet from 'icons/SealWallet'
+import Twitter from 'icons/Twitter'
 import WalletStore from 'stores/WalletStore'
 import classnames, {
   alignItems,
+  backgroundColor,
+  borderWidth,
   cursor,
   display,
+  height,
   lineHeight,
   space,
   textAlign,
   width,
 } from 'classnames/tailwind'
 import getEtherscanAddressUrl from 'helpers/getEtherscanAddressUrl'
+import useBreakpoints from 'hooks/useBreakpoints'
 
 const walletContainer = classnames(
   display('inline-flex'),
   alignItems('items-center'),
   space('space-x-4'),
   cursor('cursor-pointer')
 )
+const accountLinkContainer = classnames(
+  display('inline-flex'),
+  alignItems('items-center'),
+  space('space-x-4'),
+  cursor('cursor-pointer')
+)
 const walletAccount = classnames(
   textAlign('text-right'),
   lineHeight('leading-5'),
   display('sm:flex', 'hidden')
 )
+const socialContainer = classnames(
+  display('inline-flex'),
+  alignItems('items-center'),
+  space('space-x-4')
+)
+const delimeterContainer = classnames(
```

Comment:
If you want to create a separator line, you can use [`tailwind` solution](https://tailwindcss.com/docs/divide-style) as well (try it, if it won't work easy, let's stay with this one)